core/src/main/kotlin/io/customer/sdk/events/Metric.kt

```diff
@@ -0,0 +1,12 @@
+package io.customer.sdk.events
+
+/**
+ * Metric enum class to represent the metrics that can be tracked for events
+ * like push, in-app, etc.
+ */
+enum class Metric {
+    Delivered,
+    Opened,
+    Converted,
```

Comment:
We aren;t using `Converted`? anywhere?

Reply:
Yup, we aren't using it anywhere moment. But I kept it assuming it might have exposed previously for customers to utilize.